website/docs/contributing/general/ADR/domain-language.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "ADR: Domain language"
+---
+
+## Background
+
+In the codebase, we have seen the need to define a domain language that we use to refer to features, methods to keep it consistent across the codebase. This ADR will contain a growing list of domain language used to keep the consistency across the codebase.
+
+## Decision
+
+We have decided to use the same domain language for the features we develop. Each feature will have it's own domain language to keep it consistent across the codebase.
+
+## Change requests domain language
+
+* *Change request*
+
+An entity referring to the overarching data structure of a change request. A change request contains changes, and can be approved or rejected.
+
+* *Changes*
+
+An entity referring to a single change within a change request
+
+* *Discard*
+
+A naming convention used for deleting a single change of a change request, or discarding an entire change request
+
+* *Pending*
+
+A naming convention used to describe a change requests state. A pending change request is in one of these three states: `Draft | In review | Approved`
```

Comment:
I sort of agree with @dsusa72's comments on considering UX and legibility as well as the term "naming convention" Would something like this work?

```suggestion
A *pending* change request is one that has not yet been applied or discarded. In other words, it is in one of these three states:
1. `Draft`
2. `In review`
3. `Approved`
```